modules/gdpr_fields/src/GDPRCollector.php

```diff
@@ -140,13 +140,18 @@ public function getValueEntities(array &$entity_list, $entity_type, EntityInterf
 
 
     foreach ($definitions as $definition_id => $definition) {
-      list($type, $definition_entity, ,) = explode(':', $definition_id);
+      list($type, $definition_entity, $related_entity_type,) = explode(':', $definition_id);
 
       // Ignore entity revisions for now.
       if ($definition_entity == 'entity_revision') {
         continue;
       }
 
+      // Ignore links back to gdpr_task.
+      if ($related_entity_type == 'gdpr_task') {
+        continue;
+      }
+
       if ($type == 'typed_data_entity_relationship') {
```

Comment:
Cool, can you stick a @todo in here to remove the gdpr_task specific continue once we have solved the concept of 'ignore/excluded' relationships and entity types.

Reply:
Done.